games_services/lib/src/models/leaderboard_score_data.dart

```diff
@@ -1,18 +1,23 @@
+import 'package:games_services/src/models/player_data.dart';
+
 class LeaderboardScoreData {
   final int rank;
   final String displayScore;
   final int rawScore;
   final int timestampMillis;
-  final String scoreHolderDisplayName;
-  final String? scoreHolderIconImage;
+  final PlayerData scoreHolder;
+
+  // TODO: deprecate in favor of accessing PlayerData properties directly
+  // provided to maintain backwards compatibility
+  String get scoreHolderDisplayName => scoreHolder.displayName;
```

Comment:
I think we should add
`@Deprecated('Deprecate in favor of accessing PlayerData properties directly')`
annotation to both of them.
What do you think?

Reply:
Agreed. Done.